### ISSUE TITLE
DEV-2: Make the Usernames of different color

### ISSUE THREAD

**Issue by KovDimaY** [KovDimaY/SimpleChat-Socket.io#8]
At the moment all the usernames are black: 
![image](https://user-images.githubusercontent.com/26466644/37853623-c6fc95c4-2ee7-11e8-9e4d-d77b465bb168.png)

Would be fun to add a mechanism, which would assign different colors to different users, so it will be easier to differentiate users in the chat.

**I propose the next two scenarios:**
1) Use random color for every new user that comes (except white and black - white is not possible to see and black would be a color of the Admin).
2) Use a fixed set of colors and assign the colors in the order as users come. It means the first user is always blue, the second is red, the third - green and so on. 

I think would be good to try both scenarios and to see which one works better.

To work on this issue, please use a branch **DEV-2** created from **devel**. When the work is finished - create a PR to the **devel** branch. In the end of each development circles, code from the **devel** branch (if everything is ok) goes to the **master** branch and deploys to production.

**Comment by KovDimaY** [comment]
I have implemented different colors for users:
![image](https://user-images.githubusercontent.com/26466644/38471311-992b1c94-3b6f-11e8-8a9f-e942ce3b2acb.png)

I decided to use pretty simple logic for this:
1) every time the new user is created, the color is assigned randomly from some fixed list of colors
2) The color of Admin always is "black", any other user cannot get this color
3) At the moment I use a set of default colors for series of Highchart library except the black one: `["#7cb5ec", "#90ed7d", "#f7a35c", "#8085e9", "#f15c80", "#e4d354", "#2b908f", "#f45b5b", "#91e8e1"]`

In theory it is possible that for two users in the same room the color will be the same even if they are the only two users in the room, but while I was testing, it never happened. Maybe in the future we could add more colors in the pallete or create more complex logic to avoid this, but at the moment I feel pretty comfortable with the result that I got. I am going to create a PR for this fix.
